### PR TITLE
Add and fix Chinese translations 2022.10.14

### PR DESCRIPTION
**Pull request by Bill-Haku**
- added translations for new words
- corrected some mistakes including Chinese & English
- add hundreds of translations of existing words (mainly in `events.json5`)

### CODE REVIEW

**Comment by xicri** [comment]
Thanks, I'll review this PR tomorrow.

File: dataset/dictionary/locations.json5 [hunk 1]
```diff
@@ -186,7 +186,7 @@
     tags: [ "mondstadt", "location" ],
   },
   {
-    en: "Stormterror's Liar",
+    en: "Stormterror's Lair",
```
Comment (xicri) [inline dataset/dictionary/locations.json5:189]: Oops, this was a typo.

When English is changed, the permalink for this word is changed at the same time.
Can you add the redirect setting to [redirect/words.json](https://github.com/xicri/genshin-langdata/blob/main/dataset/redirect/words.json)?

Current URL is https://genshin-dictionary.com/ja/stormterrors-liar/, so you can configure the redirect by adding the following line:

```json
"stormterrors-liar": "stormterrors-lair"
```

File: dataset/dictionary/y-events.json5 [hunk 2]
```diff
@@ -1,12 +1,14 @@
 [
   {
-    en: "Theater Machanicus",
+    en: "Theater Mechanicus",
```
Comment (xicri) [inline dataset/dictionary/y-events.json5:3]: Can you add the redirect setting to [redirect/words.json](https://github.com/xicri/genshin-langdata/blob/main/dataset/redirect/words.json) as well?

The current URL is https://genshin-dictionary.com/ja/theater-machanicus/, so you can configure the redirect by adding the following line:

```json
"theater-machanicus": "theater-mechanicus"
```

File: dataset/dictionary/y-events.json5 [hunk 3]
```diff
@@ -155,50 +176,58 @@
...
     ja: "五風巡察",
+    zhCN: "循察五风",
     pronunciationJa: "ごふうじゅんさつ",
     notes: "v3.1 期間限定イベント",
     tags: [ "event" ],
   },
   {
     en: "Weinlesefest",
     ja: "ブリーズブリュー祭",
+    zhCN: "归风佳酿节",
```
Comment (xicri) [inline dataset/dictionary/y-events.json5:195]: In the Japanese version, this festival is called ブリーズブリュー祭 in the v3.1 PV, but it is actually called ブリュー祭 in the game. Both are called Weinlesefest in English version.
Is there such a different name in the Chinese version? I will add ブリュー祭 later, so if there is the corresponding word to ブリュー祭 in Chinese, I want to add it.

File: dataset/dictionary/y-events.json5 [hunk 4]
```diff
@@ -356,52 +391,60 @@
...
     en: "Hues of the Violet Garden",
     ja: "華やぐ紫苑の庭",
+    zhCN: "堇庭华彩",
     pronunciationJa: "はなやぐしおんのにわ",
     notes: "v2.6 期間限定イベント",
     tags: [ "event" ],
   },
   {
-    en: "Magnificent Irodori Festival",
+    en: "Maganificent Irodori Festival",
```
Comment (xicri) [inline dataset/dictionary/y-events.json5:400]: According to the official PV, "Magnificent" seems correct.
https://youtu.be/LqCwQicfMuc?t=165

```suggestion
    en: "Magnificent Irodori Festival",
```

File: dataset/dictionary/y-events.json5 [hunk 5]
```diff
@@ -727,20 +810,23 @@
...
     en: "Phantom Flow",
     ja: "幻影心流",
+    zhCN: "幻影心流",
     pronunciationJa: "げんえいしんりゅう",
     notes: "v2.0 期間限定イベント",
     tags: [ "event" ],
   },
   {
-    en: "Theater Machanicus: Stage of Wonders",
+    en: "Theater Mechanicus: Stage of Wonders",
```
Comment (xicri) [inline dataset/dictionary/y-events.json5:827]: Can you add the redirect setting to [redirect/words.json](https://github.com/xicri/genshin-langdata/blob/main/dataset/redirect/words.json) as well?

The current URL is https://genshin-dictionary.com/ja/theater-machanicus-stage-of-wonders/, so you can configure the redirect by adding the following line:

```json
"theater-machanicus-stage-of-wonders": "theater-mechanicus-stage-of-wonders"
```

**Review by xicri — CHANGES_REQUESTED** [review]

File: dataset/dictionary/characters.json5 [hunk 6]
```diff
@@ -2059,6 +2067,7 @@
   {
     en: "King Deshret",
     ja: "キングデシェレト",
+    // According to this [wiki](https://genshin-impact.fandom.com/wiki/Scarlet_King), it shares the same name in Chinese with the Scarlet King.
```
[the remark this answers is not in the document]
Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:2070]: fine

File: dataset/dictionary/locations.json5 (hunk 1 above)
Comment (Bill-Haku) [reply dataset/dictionary/locations.json5:189]: Just add this line after `tags: [ "..." ]` right? I will do it right now.

File: dataset/dictionary/y-events.json5 (hunk 3 above)
Comment (Bill-Haku) [reply dataset/dictionary/y-events.json5:195]: Yes, it is 佳酿节 in Chinese.

File: dataset/dictionary/y-events.json5 (hunk 4 above)
Comment (Bill-Haku) [reply dataset/dictionary/y-events.json5]: You are right. I checked the dictionary and it seems that I have been spelling this word wrong for a long time LOL.

File: dataset/dictionary/locations.json5 (hunk 1 above)
Comment (xicri) [reply dataset/dictionary/locations.json5:189]: Ah, no, you need to add the line in another file: [redirect/words.json](https://github.com/xicri/genshin-langdata/blob/main/dataset/redirect/words.json)

Comment (Bill-Haku) [reply dataset/dictionary/locations.json5:189]: Oh, OK.

File: dataset/dictionary/y-events.json5 (hunk 3 above)
Comment (xicri) [reply dataset/dictionary/y-events.json5:195]: OK, then let's make like this:

```suggestion
    zhCN: "佳酿节 / 归风佳酿节",
```

**Comment by xicri** [comment]
Recently, Cloudflare Pages sometimes deploys branches of the past PRs...
Just ignore the above comment.

**Pull request merged**